### PR TITLE
add lesson_04

### PR DESCRIPTION
**Pull request by kojalowicz**

### CODE REVIEW

**Comment by kojalowicz** [comment]
@zamber prosze o review

File: students/kojalowicz_piotr/lesson_04/test_for_fibonacci_numbers.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                return fib(n)
+            else:
+                print('The number must integer biger then -1')
+                n = int(input())
+        except ValueError:
+            print('The number must integer biger then -1')
+            n = int(input())
+
+
+class MyTest(unittest.TestCase):
```
Comment (zamber) [inline students/kojalowicz_piotr/lesson_04/test_for_fibonacci_numbers.py]: This could be extended with type edge case and/or performance tests.
